third_party/xla/xla/stream_executor/rocm/rocm_driver.cc

```diff
@@ -1465,7 +1465,7 @@ GpuDriver::GraphNodeGetType(hipGraphNode_t node) {
   hipDeviceProp_t props;
   hipError_t result = wrap::hipGetDeviceProperties(&props, device);
   if (result == hipSuccess) {
-    *version = props.gcnArch;
+    *version = props.gcnArchName;
```

Comment:
is it ok to add rocm version check here?

Reply:
> is it ok to add rocm version check here?

If an older ROCm version that we still support does not have gcnArchName in it, then we should probably add a ROCm version check.

Reply:
I dont think we need a check - this API has been there since. The change is dumping the arch specific one.
This code is all wrong though. The old API uses int, the new one is a string. Will fix